concent_api/verifier/utils.py

```diff
@@ -0,0 +1,47 @@
+from base64 import b64encode
+import os
+import requests
+
+from django.conf import settings
+
+from golem_messages import message
+from golem_messages.shortcuts import dump
+
+
+def clean_directory(directory_path: str):
+    """ Removes all files from given directory path. """
+    for file in os.listdir(directory_path):
+        file_path = os.path.join(directory_path, file)
+        try:
+            if os.path.isfile(file_path):
+                os.unlink(file_path)
+        except OSError:
```

Comment:
I would suggest adding some logging here - we should know if for some reason file is not deleted